getting-started/vs-code-integration/new-project-wizard.md

```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
 
 1. Type/Select `Telerik UI for Blazor Template Wizard: Launch` and press `Enter` to launch the extension.
 
-## Configure the Project
+## Creating the Project
 
 The **Create New Project** wizard provides two options to start you project:
 
 * **Start from Blank Project** - allows you to create a blank project that is pre-configured for the Progress® Telerik® UI for Blazor components. You can select and add sample pages to get started (a Grid with CRUD operations form a service, a Form, Charts).
 
-* **Start from a Template** - the wizard provides an Admin Dashboard Template - a fully functional sample app that showcases some of the most popular Telerik UI for Blazor components, such as Grid, TileLayout, Form, Charts, Gauges, etc. in a real world scenario.
+* **Start from a Template** - the wizard provides a set of predefined project templates:
+    * **CRUD, Form, Chart**&mdash;A small app that showcases the Telerik UI Data Grid, Chart, and forms validation. The Grid also uses a basic CRUD service which is a common pattern for data updates in real applications.
```

Comment:
* Should it be `Form validation` instead?
* For the second sentence, consider going with `The Data Grid also uses a CRUD service (a common pattern for data updates in production-ready applications).`